File: python_eclipse/HELLO_PYTHON/day03/ooptest.py
class Animal:
    def __init__(self):
        print("생성자")
        self.age = 0
        
    def getOlder(self):
        self.age += 1
    
    def __del__(self):
        print("소멸자")
    
    

class Human(Animal):
    def __init__(self):
        super().__init__()
        self.language = 0
        
    def momstouch(self,stroke):
        self.language += stroke
        

    

if __name__  == '__main__' :
    hum = Human() 
    print(hum.language)
    print(hum.age)
    hum.getOlder()
    hum.momstouch(5)
    print(hum.language)
    print(hum.age)
    
    

File: python_eclipse/HELLO_JAVA/src/day02/OopTest.java
package day02;

public class OopTest {
	public static void main(String[] args) {
		
		/*Animal ani = new Animal();
		System.out.println(ani.age);
		ani.getOlder();
		System.out.println(ani.age);*/
		
		Human hum = new Human();
		System.out.println(hum.age);
		System.out.println(hum.language);
		hum.getOlder();
		hum.momstouch(7);
		System.out.println(hum.age);
		System.out.println(hum.language);
		
		
	}
}
